converter/include/k4SimDelphes/DelphesEDM4HepConverter.h

```diff
@@ -40,6 +41,11 @@ namespace k4SimDelphes {
    */
   constexpr std::array<std::string_view, 1> RECO_CLUSTER_OUTPUT = {"Tower"};
 
+  /**
+   * Classes that will be stored as TrackerHits
+   */
+  constexpr std::string_view TRACKERHIT_OUTPUT_NAME = "TrackerHits"; 
```

Comment:
```suggestion
  constexpr std::string_view TRACKERHIT_OUTPUT_NAME = "TrackerHits";
```

clang-tidy is complaining about this. Trying to fix it via a commit here.